packages/common/src/store/playback-position/types.ts

```diff
@@ -0,0 +1,14 @@
+import { ID } from 'models/Identifiers'
+
+export const PLAYBACK_POSITION_LS_KEY = 'playbackPosition'
+
+export type PlaybackStatus = 'IN_PROGRESS' | 'COMPLETED'
+
+export type PlaybackPositionInfo = {
+  status: PlaybackStatus
+  playbackPosition: number
+}
+
+export type PlaybackPositionState = {
+  trackPositions: Record<ID, PlaybackPositionInfo>
```

Comment:
so one thing i learned with Record, is that trackPositions[1] will always return PlaybackPositionInfo, which is technically not true, so maybe we should do 
```
trackPositions: {[Key in ID]?: PlaybackPositionInfo}
```

Reply:
Oh interesting, good tip here